qa/tasks/cephfs/test_admin.py

```diff
@@ -187,6 +187,37 @@ def gen_health_warn_mds_trim(self):
 
         self.wait_for_health(health_warn, 30)
 
+    def run_dump_stray_command(self):
```

Comment:
For this to work, it should be called from a test function.
You should probably move this to another class and prefix `test_` with the function name.

Reply:
OH thanks for that! Move to another file - I was thinking that this is ASOK command to it should be placed there. But I am ok to move it to the test_strays.py